### DISCUSSION TITLE
Android apk?

### DISCUSSION THREAD

**Discussion by beng7** [claudiodangelis/qr-filetransfer#242] [Ideas]
I'm opening this issue because:

- [ x] I want to request a feature

Hi
Any chance of an android version please?
Thanks

**Comment by claudiodangelis** [comment]
Hello, I think the concept of a disposable web server reachable through QR code could be ported to a mobile platform, however, I don't have the skills for developing an Android app.
Thanks for the idea!

**Comment by leestarb** [comment]
Is this possible without rewriting the project in Java? And even Java will require other technologies to be used. But there may be a Go-to-Java(Android) solution (like python-for-android) allowing to write android apps with Go and its modules!

**Comment by claudiodangelis** [comment]
Hello, maybe you can try out this app by @ianfixes : https://github.com/ianfixes/qrcp-android

I did not try it personally yet.
C

**Comment by ianfixes** [comment]
As of this posting, my [Android clone of qrcp](https://github.com/ianfixes/qrcp-android
) is now linked from the [QRCP README](https://github.com/claudiodangelis/qrcp?tab=readme-ov-file#clones-and-similar-projects).  

Check the [releases page](https://github.com/ianfixes/qrcp-android/releases) to install it (I'm not in the App store) or build it on your own machine.

I don't make use of any of the same code, as the core concepts (HTTP servers and QR codes) are provided by dozens of free libraries and all that was needed was to stitch them together.  Most of the Android logic is spent reacting to the state of the network and handling file permissions.
